jpsonic-main/src/main/java/com/tesshu/jpsonic/service/upnp/processor/AlbumByGenreUpnpProcessor.java

```diff
@@ -147,26 +138,24 @@ public MediaFile getItemById(String id) {
 
     @Override
     public int getChildSizeOf(MediaFile item) {
-        return searchService.getAlbumsByGenres(item.getName(), 0, Integer.MAX_VALUE, util.getGuestMusicFolders())
-                .size();
+        return searchService.getAlbumsByGenres(item.getName(), 0, Integer.MAX_VALUE, util.getGuestFolders()).size();
```

Comment:
Add count method to Dao.

Reply:
Ah.... This was implemented in Lucene. And it is assumed that it is cached and the correct offset count is used. We can leave it as is unless it causes a fatal problem.